packages/ui/src/ui-component/input/Input.js

```diff
@@ -48,6 +48,12 @@ export const Input = ({ inputParam, value, nodes, edges, nodeId, onChange, disab
         }
     }, [myValue])
 
+    useEffect(() => {
+        if (value) {
```

Comment:
why this changes here @0xi4o

Reply:
This is same as the `AsyncDropdown`. It works without the `useEffect`.